test_crates/src/test_cases/struct_repr_c_removed.rs

```diff
@@ -1,10 +1,35 @@
 #[cfg(not(feature = "struct_repr_c_removed"))]
 #[repr(C)]
-pub struct Foo {
+pub struct CRemoved {
     pub bar: usize,
 }
 
 #[cfg(feature = "struct_repr_c_removed")]
-pub struct Foo {
+pub struct CRemoved {
     pub bar: usize,
 }
+
+#[cfg(not(feature = "struct_repr_c_removed"))]
+#[repr(C, align(16))]
+pub struct CRemovedAlign16 {
+    pub bar: usize,
+}
+
+#[cfg(feature = "struct_repr_c_removed")]
+#[repr(align(16))]
+pub struct CRemovedAlign16 {
+    pub bar: usize,
+}
+
+#[cfg(not(feature = "struct_repr_c_removed"))]
+#[repr(C)]
+#[repr(align(16))]
+pub struct SeparateCAlign16 {
+    pub bar: usize,
+}
+
+#[cfg(feature = "struct_repr_c_removed")]
+#[repr(C, align(16))]
+pub struct SeparateCAlign16 {
+    pub bar: usize,
+}
```

Comment:
```suggestion
}

```
You should be able to configure your editor to always automatically add this terminating newline when it saves.

Reply:
It is indeed possible, and very easy to do - [here](https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files). Thanks for suggesting that such features exist!